eng/performance/benchmark_jobs.yml

```diff
@@ -92,9 +93,9 @@ jobs:
             ${{ each framework in parameters.frameworks }}:
               ${{ framework }}:
                 _Framework: ${{ framework }}
-                _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
+                _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}${{ parameters.libraryVersion }}"
```

Comment:
I don't know how the new uploader works so I've just extended the `RunKind` string with `fixed|rolling` This parameter is optional and null by default, so the "id" should change only for ML.NET runs

Reply:
This seems like the right way to go about this.